1.10.12/alpine3.10/Dockerfile

```
@@ -21,12 +21,13 @@ ARG VERSION="1.10.12-2.*"
 ARG SUBMODULES="all, statsd, elasticsearch"
 ARG AIRFLOW_MODULE="astronomer_certified[${SUBMODULES}]==$VERSION"
 ARG REPO_BRANCH=master
+ENV AIRFLOW_VERSION="1.10.12"
```

```suggestion
ARG AIRFLOW_VERSION="1.10.12"
```

Env is visible in the resulting docker image, but it doesn't need to be.

---

The only reason I did that was so no one can overwrite the Airflow Version during docker-build time.

I can change that to using ARG but just was overly-cautious :D

---

updated in https://github.com/astronomer/ap-airflow/pull/160/commits/28339144b747aadea11ed8b215697b8acc4b805d